.github/CONTRIBUTING.md

```diff
@@ -56,6 +56,10 @@ $ cd PeerTube
 $ yarn install --pure-lockfile
 ```
 
+Note that development is done on the `develop` branch. If you want to hack on
```

Comment:
Develop isn't really made on develop branch, in fact you should durring work on an other branch and merge into develop when finish.

Reply:
I agree but to keep it as simple as possible, we don't really want to specify how the developer should work. If they want to use the git flow, they can, if they want to work directly on develop on their fork, they can too.